common/views/components/ZoomedImage/ZoomedImage.tsx

```diff
@@ -85,6 +95,7 @@ const ZoomedImage = ({ id, infoUrl, setShowViewer, isFullscreen }: Props) => {
             },
           ],
         });
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Could we just remove the `_`?

Reply:
actually removing this should be fine . !